### ISSUE TITLE
Details about train in SHHS dataset

### ISSUE THREAD

**Issue by yudaleng** [leoaoli1/DeepCAD#3]
Hi,

While I was reproducing your training code for the SHHS dataset, I ran into some problems and would like to ask you for advice. I hope I can get your help:

1. what are the recommended values for the hyperparameters n_epochs, n_batch, lr and weight_decay during training?
2. how is the labeling handled when converting the SHHS dataset from EDF format to HDF format? Are labels generated on a per second (1s) basis?
3. was the data truncated during the conversion from EDF to HDF format? If so, how did you ensure that the labels were the same length as the ECG data? For example, how is len(ecg)//256 == len(label) ensured?
4. If it is convenient, can you provide the code you used to convert EDF to HDF format?

Looking forward to your reply, thank you very much!
